gwin/likelihood.py

```diff
@@ -28,22 +28,126 @@
 """
 
 import numpy
+import logging
 
 from scipy import (stats, special)
 
-from pycbc import (conversions, filter, transforms)
-from pycbc.waveform import NoWaveformError
+from pycbc import (conversions, filter, transforms, distributions)
+from pycbc.waveform import (generator, NoWaveformError)
 from pycbc.types import Array
 from pycbc.io import FieldArray
+from pycbc.workflow import ConfigParser
 
 # Used to manage a likelihood instance across multiple cores or MPI
 _global_instance = None
 
-
 def _call_global_likelihood(*args, **kwds):
     return _global_instance(*args, **kwds)  # pylint:disable=not-callable
 
 
+# XXX: import the following from pycbc.distributions once PR #2123 has made
```

Comment:
PR #2123 has been merged into the latest release, so we could just import it.

Reply:
@vivienr Done. @duncanmmacleod Since this requires the latest release of pycbc, I've added version requirements for pycbc to `setup.py` and `requirements.txt`.